pkg/executor/backoff_test.go

```diff
@@ -45,42 +45,53 @@ var _ = Describe("exponential limited backoff", func() {
 		testsClock.Step(time.Nanosecond)
 	})
 
-	It("should be ready before first step", func() {
+	It("should be ready and not timeout before first step", func() {
 		Expect(backoff.Ready()).To(BeTrue())
+		Expect(backoff.IsTimeout()).To(BeFalse())
 	})
 
 	It("should not be ready before step end time", func() {
 		Expect(backoff.Ready()).To(BeTrue())
+		Expect(backoff.IsTimeout()).To(BeFalse())
 		backoff.Step()
 		Expect(backoff.Ready()).To(BeFalse())
+		Expect(backoff.IsTimeout()).To(BeFalse())
 	})
 
-	It("should be ready after step end time passed", func() {
+	It("should be ready and not timeout after step end time passed", func() {
 		firstStepDuration := duration + time.Nanosecond
 		secondStepDuration := time.Duration(float64(firstStepDuration)*factor) + time.Nanosecond
 
 		Expect(backoff.Ready()).To(BeTrue())
+		Expect(backoff.IsTimeout()).To(BeFalse())
 		backoff.Step()
 
 		Expect(backoff.Ready()).To(BeFalse())
+		Expect(backoff.IsTimeout()).To(BeFalse())
 		testsClock.Step(firstStepDuration)
 
 		Expect(backoff.Ready()).To(BeTrue())
+		Expect(backoff.IsTimeout()).To(BeFalse())
 		backoff.Step()
 
 		Expect(backoff.Ready()).To(BeFalse())
+		Expect(backoff.IsTimeout()).To(BeFalse())
 		testsClock.Step(secondStepDuration)
 
 		Expect(backoff.Ready()).To(BeTrue())
+		Expect(backoff.IsTimeout()).To(BeFalse())
 	})
 
-	It("should not be ready after max time is passed", func() {
+	It("should not be ready and not timeout after max time is passed", func() {
```

Comment:
I think the existing description is accurate, I see no reason to add anything else. BTW, what was added is wrong, the timeout has been reached.

Reply:
good catch :) 

regarding keeping the existing description:
- since `ready` is not `timeout` logic-wise - should we not also mention that?
- do you mean on all tests?